jsonld2compactjsonldldj/jsonld2compactjsonldldj.py

```diff
@@ -15,52 +43,52 @@ def run():
     optional_arguments = parser._action_groups.pop()
 
     required_arguments = parser.add_argument_group('required arguments')
-    required_arguments.add_argument('-context', type=str, help='The JSON-LD context file', required=True)
+    required_arguments.add_argument('-context', type=str, help='The JSON-LD context file')
```

Comment:
removing the "required" option here, will not lead to a removal of this argument from the list of required arguments (in the help). Hence, it might be a bit misleading. I would recommend to completely remove the argument '-context' and shift the argument '-context-url' to the list of required arguments + mark this argument really as 'required' + enable and/or document the processing that of this argument that it is also possible to provide local files (e.g. via file:///....).